### PR TITLE
set normal bounds to normal bounds width

### PR DESCRIPTION
**Pull request by akdetrick**
fixed Sass var for normal bounds max-width

### CODE REVIEW

File: assets/scss/components/_bounds.scss [hunk 1]
```diff
@@ -13,12 +13,12 @@ padding, and sets a max-width.
 	box-sizing: border-box;
 	margin-left: auto;
 	margin-right: auto;
-	max-width: $bounds-wide;
+	max-width: $bounds;
```
Comment (eilinora) [inline assets/scss/components/_bounds.scss]: i thought we wanted bounds wide, and then perhaps there is a prop add to bounds narrow

Comment (nlaz) [inline assets/scss/components/_bounds.scss]: There's still a [`bounds--wide` class](https://github.com/meetup/meetup-web-components/pull/132/files/5173cbacc24b7998d180728ef8449455e3f00127#diff-887b5fa95ea8cd21b4b4f2d18e1ae9d2R23). But I don't think our `Bounds` components supports that as a prop yet.

Comment (eilinora) [inline assets/scss/components/_bounds.scss]: true, but if the goal is to use the bounds component feels like it should be an optional param there

Comment (akdetrick) [inline assets/scss/components/_bounds.scss]: Using an optional prop instead. (see updated diff)

**Pull request merged**